parquet-format-structures/pom.xml

```diff
@@ -0,0 +1,184 @@
+  - or more contributor license agreements.  See the NOTICE file
+  - distributed with this work for additional information
+  - regarding copyright ownership.  The ASF licenses this file
+  - to you under the Apache License, Version 2.0 (the
+  - "License"); you may not use this file except in compliance
+  - with the License.  You may obtain a copy of the License at
+  -
+  -   http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing,
+  - software distributed under the License is distributed on an
+  - "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  - KIND, either express or implied.  See the License for the
+  - specific language governing permissions and limitations
+  - under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.parquet</groupId>
+    <artifactId>parquet</artifactId>
+    <relativePath>../pom.xml</relativePath>
+    <version>1.10.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>parquet-format-structures</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Apache Parquet Format Structures</name>
+  <url>http://parquet.apache.org/</url>
+  <description>Parquet-mr related java classes to use the parquet-format thrift structures.</description>
+
+  <properties>
+    <parquet.thrift.path>${project.build.directory}/parquet-format-thrift</parquet.thrift.path>
+  </properties>
+
+  <build>
+    <plugins>
+      <!-- Getting the parquet-format thrift file -->
+       <plugin>
+         <groupId>org.apache.maven.plugins</groupId>
+         <artifactId>maven-dependency-plugin</artifactId>
+         <executions>
+           <execution>
+             <id>unpack</id>
+             <phase>generate-sources</phase>
+             <goals>
+               <goal>unpack</goal>
+             </goals>
+             <configuration>
+               <artifactItems>
+                 <artifactItem>
+                   <groupId>org.apache.parquet</groupId>
+                   <artifactId>parquet-format</artifactId>
+                   <version>${parquet.format.version}</version>
+                   <type>jar</type>
+                 </artifactItem>
+               </artifactItems>
+               <includes>parquet.thrift</includes>
+               <outputDirectory>${parquet.thrift.path}</outputDirectory>
+             </configuration>
+           </execution>
+         </executions>
+       </plugin>
+      <!-- thrift -->
+      <plugin>
+        <groupId>org.apache.thrift.tools</groupId>
+        <artifactId>maven-thrift-plugin</artifactId>
+        <version>0.1.11</version>
+        <configuration>
+          <thriftSourceRoot>${parquet.thrift.path}</thriftSourceRoot>
+          <thriftExecutable>${thrift.executable}</thriftExecutable>
```

Comment:
Should this be `format.thrift.executable`? `thrift.executable` is used by the parquet-thrift module and we want to be able to set it separately.